Jack/Gen.fs

```diff
@@ -17,8 +17,8 @@ module Gen =
     let create (shrink : 'a -> LazyList<'a>) (random : Random<'a>) : Gen<'a> =
         Random.map (Tree.unfold id shrink) random |> ofRandom
 
-    let result (x : 'a) : Gen<'a> =
-        Tree.result x |> Random.result |> ofRandom
+    let constant (x : 'a) : Gen<'a> =
+        Tree.singleton x |> Random.constant |> ofRandom
```

Comment:
Indeed. And other candidate names could be `inject`, `retn` (`return` is a reserved word), and so on.